modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -215,6 +215,9 @@ def prepare_appointment(appointment)
         appointment[:requested_periods] = nil if booked?(appointment) && VAOS::AppointmentsHelper.cerner?(appointment)
 
         convert_appointment_time(appointment)
+
+        appointment[:station], appointment[:ien] = *extract_station_and_ien(appointment)&.first(2)
```

Comment:
Also, not sure if I did this in the best way. I noticed that station and ien were both being called elsewhere in the service, so I figured it would be best to just set those values on the appointment itself, rather than call `extract_station_and_ien` twice